src/main/java/org/folio/search/service/IndexService.java

```diff
@@ -56,6 +61,9 @@ public FolioCreateIndexResponse createIndex(String resourceName, String tenantId
   public FolioIndexOperationResponse updateMappings(String resourceName, String tenantId) {
     var index = getElasticsearchIndexName(resourceName, tenantId);
     var mappings = mappingHelper.getMappings(resourceName);
+
+    log.info("Updating mappings for resource [{}] for tenant [{}] - [{}]",
```

Comment:
log format always anywhere in search should be like ```$message [$parameter: $value]```

Reply:
thanks, changed.